pandas/core/frame.py

```diff
@@ -3161,6 +3167,21 @@ def query(self, expr, inplace=False, **kwargs):
         For further details and examples see the ``query`` documentation in
         :ref:`indexing <indexing.query>`.
 
+        *Backtick quoted variables*
+
+        Backtick quoted variables are parsed as literal Python code and
+        are converted internally to a Python valid identifier.
+        This can lead to the following problem.
+
+        The parsing can result in an error for certain characters.
+        These characters are those that fall outside the ASCII range
```

Comment:
are there sufficient examples below?

Reply:
I looked a bit more in depth to this, and made it a bit more detailed. I also looked into the lexical analysis document of Python and I think there are no more difficult cases as described here.